src/components/structures/RoomStatusBar.tsx

```diff
@@ -306,6 +335,37 @@ export default class RoomStatusBar extends React.PureComponent<IProps, IState> {
             return this.getUnsentMessageContent();
         }
 
+        if (this.state.timelineNeedsRefresh) {
+            return (
+                <div className="mx_RoomStatusBar mx_RoomStatusBar_unsentMessages">
+                    <div role="alert">
+                        <div className="mx_RoomStatusBar_unsentBadge">
+                            <img
+                                src={require("../../../res/img/feather-customised/warning-triangle.svg").default}
+                                width="24"
+                                height="24"
+                                title="/!\ "
+                                alt="/!\ " />
```

Comment:
this is not human-readable.

Reply:
This is the same as the existing warning icon we use for lost connection. We should update both 👍

https://github.com/matrix-org/matrix-react-sdk/blob/c35fc169f532e3bd9e2a54cdef382ae8570f08f6/src/components/structures/RoomStatusBar.tsx#L285-L293

<img width="701" alt="" src="https://user-images.githubusercontent.com/558581/163341691-76a83651-8a72-4f34-91af-ff685ff63e34.png">